CONTRIBUTING.md

```diff
@@ -11,19 +11,18 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
 
 ## Requesting An Icon
 
-We welcome icon requests. However, before you submit a new issue please make sure the icon:
+We welcome icon requests, before you submit a new issue please make sure the icon:
 
 * Is of a _popular_ brand.
 * Has not already been requested.
-* Can be represented in a monochromatic format.
```

Comment:
We've had issues with this in the past, see Sendgrid. But I understand that this may not be the most clear to a new contributor.

Reply:
I agree that it can cause issues, but I think it is better to have the maintainers judge if the icon is monochromatic. Take e.g. [Redis](https://github.com/simple-icons/simple-icons/pull/613): it is not clear if it is representable in a monochromatic format, but with some effort it is definitely possible